### ISSUE TITLE
SublimeLinter: WARNING: Aborting lint. SublimeLinter needs a restart of Sublime.

### ISSUE THREAD

**Issue by tiagoporto** [SublimeLinter/SublimeLinter#1304]
I,m using Sublime Linter with Sublimelinter-eslint on Sublime 3 on Mac OS 10.12.6

I,m getting this warning, even if the sublime was restarted.

```
SublimeLinter: WARNING: Aborting lint. SublimeLinter needs a restart of Sublime.
```

**Comment by tiagoporto** [comment]
After delete old linter files, reinstall and reload linter and plugin, works.

**Comment by Jones-S** [comment]
I am experiencing the same issue. I removed all linter packages, removed [sass-lint](https://github.com/skovhus/SublimeLinter-contrib-sass-lint) (the linter I want to use) but I still get the same warning.

If I open the console in Sublime I get this:
( I think there is other stuff in it (imagemagick) but I left it as is, to make sure I can show the whole log)

```
…

Error:
/bin/sh: convert: command not found

Highlighting colors with gutter icons is not supported with current ImageMagick setup. Try configuring the "icon_factory setting"
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 210, in on_api_ready
    m.plugin_loaded()
  File "sublime_linter in /Users/XXX/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package", line 47, in plugin_loaded
  File "sublime_linter in /Users/XXX/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package", line 37, in backup_old_settings
TypeError: argument of type 'NoneType' is not iterable
Emmet: No need to update PyV8
SublimeLinter: WARNING: Aborting lint. SublimeLinter needs a restart of Sublime.
SublimeLinter: WARNING: Aborting lint. SublimeLinter needs a restart of Sublime.
SublimeLinter: WARNING: Aborting lint. SublimeLinter needs a restart of Sublime.
Package Control: Skipping automatic upgrade, last run at 2018-04-18 12:26:59, next run at 2018-04-18 13:26:59 or after
environment variables loaded using: /bin/zsh -l
```

Any idea how I could fix this?

**Comment by braver** [comment]
You may need to take that up with the author of that linter. Or run in debug mode and check the output in the console for hints. 
You could also try using stylelint, it's a better linter in my experience.

**Comment by Jones-S** [comment]
ok. I was so desperate, and I downloaded atom for now. If I feel strong again I will maybe try stylelint and the debug mode. thx anyway

**Comment by braver** [comment]
LOL, that's one way to deal with it
